mocks/mocks.py

```diff
@@ -585,26 +585,27 @@ def compute_apparent_ABmag(z,
     ----------
     z : float
         The redshift of the target galaxy at the demanded distance.
-    galaxy_mass : float
-        The total mass of the target galaxy and all masses in its environment
-        in solar mass units.
-    mass_to_light : float
-        The mass to light ratio of the target.
+    particle_light : astropy.units.Quantity
+        The light of the simulation particles.
+    config : dic
+        A dictionary of configuration items.
+    cosmo : astropy.cosmology.core.FlatLambdaCDM, optional
```

Comment:
Doesn't this come packed in the `config` dictionary? Suggest removing this.

Reply:
No, it does not. But we should make a decision about how to introduce known cosmologies to the code.